Makefile

```diff
@@ -237,6 +237,7 @@ keystone_deploy_cleanup: ## cleans up the service instance, Does not affect the
 	$(eval $(call vars,$@,keystone))
 	oc kustomize ${DEPLOY_DIR} | oc delete --ignore-not-found=true -f -
 	rm -Rf ${OPERATOR_BASE_DIR}/keystone-operator ${DEPLOY_DIR}
+	oc rsh -t mariadb-openstack mysql -u root --password=${PASSWORD} -e "drop database keystone;" || true
```

Comment:
Not a concern, I'm just curious because I've used `oc exec -it  pod/mariadb-openstack -- mysql ...` so far and I wonder what difference it makes compared to using rsh

Reply:
`rsh` is a convenience function to avoid having to use `exec` with extra flags to open an interactive shell, but it can also be used for one-off commands just like `exec` can do